man/addTransform.Rd

```diff
@@ -21,12 +21,16 @@ addTransform(obj, fn, name = NULL, params = NULL, packages = NULL)
 Add a transformation function to be applied to each subset of a distributed data object
 }
 \details{
-When you add a transformation to a distributed data object, the transformation is not applied immediately, but is deferred until a function that kicks off a computation is done.  These include \code{\link{divide}}, \code{\link{recombine}}, \code{\link{drJoin}}, \code{\link{drLapply}}, \code{\link{drFilter}}, \code{\link{drSample}}, \code{drSubset}.  When any of these are invoked on an object with a transformation attached to it, the transformation will be applied in the map phase of the MapReduce computation prior to any other computation.  The transformation will also be applied any time a subset of the data is requested.  Thus although the data has not been physically transformed after a call of \code{addTransform}, we can think of it conceptually as already being transformed.
+When you add a transformation to a distributed data object, the transformation is not applied immediately, but is deferred until a function that kicks off a computation is done.  These include \code{\link{divide}}, \code{\link{recombine}}, \code{\link{drJoin}}, \code{\link{drLapply}}, \code{\link{drFilter}}, \code{\link{drSample}}, \code{drSubset}.  When any of these are invoked on an object with a transformation attached to it, the transformation will be applied in the map phase of the MapReduce computation prior to any other computation.  The transformation will also be applied any time a subset of the data is requested.  Although the data has not been physically transformed after a call of \code{addTransform}, we can think of it conceptually as already being transformed.
+
+To force the transformation to be immediately calculated on all subsets use: \code{recombine(dat, combDdo, output = ...)}.
```

Comment:
Can you change `recombine(dat, combDdo, output = ...)` to `drPersist(dat, output = ...)`?